futures-core/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+//! Core traits and types for asynchronous operations in Rust.
+
+#![no_std]
+#![deny(missing_docs, missing_debug_implementations)]
+#![doc(html_root_url = "https://docs.rs/futures/0.2")]
+
+#[macro_use]
+#[cfg(feature = "std")]
+extern crate std;
+
+#[doc(hidden)]
+#[macro_export]
```

Comment:
Since this is so tiny, could it be duplicated where needed?

Reply:
Done.